staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -81,12 +108,12 @@ type Event struct {
 	// merging. It is an external versioned object type, and may not be a valid object on its own.
 	// Omitted for non-resource requests.  Only logged at Request Level and higher.
 	// +optional
-	RequestObject runtime.Unknown
+	RequestObject *runtime.Unknown
 	// API object returned in the response, in JSON. The ResponseObject is recorded after conversion
 	// to the external type, and serialized as JSON.  Omitted for non-resource requests.  Only logged
 	// at Response Level.
 	// +optional
-	ResponseObject runtime.Unknown
+	ResponseObject *runtime.Unknown
```

Comment:
Actually, there's no precedent for using `runtime.Unkown` directly, so I changed to using a pointer type here /cc @sttts